uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -214,52 +214,8 @@ public function upgrade_0006() {
    * @return bool
    */
   public function upgrade_0007() {
-    $optionGroupID = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_OptionGroup', 'ta_settings', 'id', 'name');
-
-    if (!$optionGroupID) {
-      $params = array(
-        'name' => 'ta_settings',
-        'title' => 'Tasks and Assignments settings',
-        'is_active' => 1,
-        'is_reserved' => 1,
-      );
-
-      civicrm_api3('OptionGroup', 'create', $params);
-
-      $optionsValue = array(
-        'documents_tab' => array(
-          'label' => 'Show or hide the Documents tab',
-          'value' => 1,
-        ),
-        'keydates_tab' => array(
-          'label' => 'Show or hide the Key Dates tab',
-          'value' => 1,
-        ),
-        'add_assignment_button_title' => array(
-          'label' => 'Configure \'Add Assignment\' button title',
-          'value' => '',
-        ),
-        'number_of_days' => array(
-          'label' => 'No of days prior to Key Date to create task',
-          'value' => 30,
-        ),
-        'auto_tasks_assigned_to' => array(
-          'label' => 'Auto generated Tasks assigned to',
-          'value' => '',
-        ),
-      );
-
-      foreach ($optionsValue as $key => $value) {
-        $opValueParams = array(
-          'option_group_id' => 'ta_settings',
-          'name' => $key,
-          'label' => $value['label'],
-          'value' => $value['value'],
-        );
-
-        civicrm_api3('OptionValue', 'create', $opValueParams);
-      }
-    }
+    // NOOP
```

Comment:
The reason for removing the code is clear, I'm curious in cases like this if we should just remove the upgrade function completely. It should be clear from the gap in numbers that something was removed. 

On one hand we have something similar to commented code (empty functions), on the other hand we keep sequential numbers for upgrader functions and have a comment to explain the reason.

I don't mind either way we do this, just wondering what your opinion on it is.